src/components/Kv/KvCarousel.vue

```diff
@@ -192,21 +192,14 @@ export default {
 		}
 	},
 	methods: {
-		/**
-		 * Move forward one slide
-		 *
-		 * @public This is a public method
-		 */
-		advance() {
-			this.goToSlide(this.nextIndex);
-		},
-		/**
-		 * Move backward one slide
-		 *
-		 * @public This is a public method
+		/** TODO
+		 * The index of the carousel after the advance carousel button is clicked
+		 * @event advance-carousel
+		 * @type {Event}
 		 */
-		previous() {
-			this.goToSlide(this.previousIndex);
+		handleUserInteraction(index, interactionType) {
+			this.goToSlide(index);
+			this.$emit('interact-carousel', interactionType);
```

Comment:
It'd be good to add a JsDoc comment here documenting this event for Storybook. Something like
				/**
				 * Fires when the user interacts with the carousel. Contains the interaction type (swipe-left, click-left-arrow, etc.) 
				 * @event interact-carousel
				 * @type {Event}
				 */